x/interchainstaking/keeper/receipt.go

```diff
@@ -224,7 +230,12 @@ func (k *Keeper) SubmitTx(ctx sdk.Context, msgs []sdk.Msg, account *types.ICAAcc
 			Memo: memo,
 		}
 
-		_, err = k.ICAControllerKeeper.SendTx(ctx, chanCap, connectionID, portID, packetData, timeoutTimestamp)
+		msg := icacontrollertypes.NewMsgSendTx(owner, connectionID, timeoutTimestamp, packetData)
```

Comment:
We can just use `portID` rather than `owner`.  In our case, the `portID` is the `owner`.

Reply:
got it

Reply:
hmmm i use the portID it return `no active channel for this owner` err when submit a tx

Reply:
i think we should use port owner as define here [https://github.com/ingenuity-build/quicksilver/blob/develop/x/interchainstaking/keeper/proposal_handler.go#L71-L92](url)

Reply:
since we establish the channel the owner was set as `portOwner` here `https://github.com/notional-labs/quicksilver/blob/sdk47-rightway/x/interchainstaking/keeper/proposal_handler.go#L74`  so i set the owner to the representive portOwner as defined in the above code